aclocal.m4

```diff
@@ -25,9 +25,11 @@ AC_SUBST(install_suffix)
 # Unwarranted assumptions:
 #   - the object file produced by AC_COMPILE_IFELSE is called "conftest.$ac_objext"
 #   - the nm(1) utility is available, and its name is "nm".
+# Here use $NM because they can cause issue in cross-compiling and because is
+# not possible use a different NM implementation (like llvm-nm)
```

Comment:
This comment contradicts the previous comment line. I would suggest something like:
#   - the nm(1) utility or an equivalent is available, and its name is defined by the $NM variable.

Reply:
Yes, this will looks better, the comment merged